libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -236,6 +237,14 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
     }
   }
 
+  private listenForIonModalWillPresent() {
+    if (this.config.interactWithBackground && this.ionModalElement) {
```

Comment:
Consider readability (and consider applying the same guarding logic to the other listenFor... methods):
```suggestion
    if (!this.ionModalElement) return;
    if (this.config.interactWithBackground ) {
```

Reply:
Fixed in next commit